source/API/CMakeLists.txt

```diff
@@ -228,9 +233,10 @@ install(
   DIRECTORY "${lib_dir}/lldb/clang"
   DESTINATION lib${LLVM_LIBDIR_SUFFIX}/lldb/)
 
+if(LLDB_ENABLE_SWIFT_SUPPORT)
```

Comment:
nit: you can merge these two conditions

```
if (LLDB_ENABLE_SWIFT_SUPPORT AND LLDB_BUILT_STANDALONE)
```

Reply:
As long as the entire block is local, that makes sense.  Otherwise, it is nice to leave the upstream code as is.

Reply:
That block is local